hetmech/test_matrix.py

```diff
@@ -1,9 +1,46 @@
 import hetio.readwrite
+import numpy
+import pytest
+from scipy import sparse
+
 from .matrix import metaedge_to_adjacency_matrix
-import numpy as np
 
 
-def test_metaedge_to_adjacency_matrix():
+def get_arrays(edge, mat_type, dtype):
+    node_dict = {
+        'G': ['CXCR4', 'IL2RA', 'IRF1', 'IRF8', 'ITCH', 'STAT3', 'SUMO1'],
+        'D': ["Crohn's Disease", 'Multiple Sclerosis'],
+        'T': ['Leukocyte', 'Lung']
+    }
+    adj_dict = {
+        'GiG': [[0, 0, 1, 0, 1, 0, 0],
+                [0, 0, 1, 0, 0, 0, 0],
+                [1, 1, 0, 1, 0, 0, 1],
+                [0, 0, 1, 0, 0, 0, 0],
+                [1, 0, 0, 0, 0, 0, 0],
+                [0, 0, 0, 0, 0, 0, 0],
+                [0, 0, 1, 0, 0, 0, 0]],
+        'GaD': [[0, 1],
+                [0, 1],
+                [1, 0],
+                [0, 1],
+                [0, 0],
+                [1, 1],
+                [0, 0]],
+        'DlT': [[0, 0],
+                [1, 0]],
+        'TlD': [[0, 1],
+                [0, 0]]
+    }
+    return node_dict[edge[0]], node_dict[edge[-1]], mat_type(adj_dict[edge],
```

Comment:
for readability let's do:

```python
row_names = node_dict[edge[0]]
col_names = node_dict[edge[-1]]
adj_matrix = mat_type(adj_dict[edge], dtype=dtype)
return row_names, col_names, adj_matrix
```

This way we improve readability while making the code document itself.